client/src/components/Workflow/Editor/Index.vue

```diff
@@ -453,20 +499,9 @@ export default {
             node.onUnhighlight();
         },
         onLint() {
-            if (this.requiresReindex) {
```

Comment:
At first I thought we don't need this because onAttemptRefactor still does https://github.com/guerler/galaxy/blob/8f3b6d83b5435b6d287849f58e2b82d3ea7780d9/client/src/components/Workflow/Editor/Index.vue#L347. But the action may reference a step id that is not stable after the change: https://github.com/guerler/galaxy/blob/8f3b6d83b5435b6d287849f58e2b82d3ea7780d9/client/src/components/Workflow/Editor/Index.vue#L357 .. so I think we still need this until we figure out how to generate some kind of stable id for a step.

Reply:
Talked about this off board. The summary is that the `hasChanges` flag covers for the node length dependent `requiresReindex` flag.